src/main/java/uk/gov/companieshouse/officerfiling/api/controller/FilingDataControllerImpl.java

```diff
@@ -30,28 +30,28 @@ public FilingDataControllerImpl(final FilingDataService filingDataService,
      * Filings.
      *
      * @param transId        the Transaction ID
-     * @param filingResource the Filing Resource ID
+     * @param filingResourceId the Filing Resource ID
      * @param request        the servlet request
      * @return List of FilingApi resources
      */
     @Override
     @GetMapping(value = "/{filingResourceId}/filings", produces = {"application/json"})
     public List<FilingApi> getFilingsData(@PathVariable("transId") final String transId,
-            @PathVariable("filingResourceId") final String filingResource,
+            @PathVariable("filingResourceId") final String filingResourceId,
             final HttpServletRequest request) {
-        final var logMap = LogHelper.createLogMap(transId, filingResource);
+        logger.debugContext(transId, "GET /private/transactions/{transId}/officers{filingId}/filings", new LogHelper.Builder(transId)
+                        .withFilingId(filingResourceId)
+                        .withRequest(request)
+                        .build());
```

Comment:
The structured logging library has a `RequestLogger` helper for logging out requests to the microservice - could save you some code maintenance:

```
requestLogger.logStartRequestProcessing(request, logger);
```

Reply:
We have a separate story to create a logging interceptor so the RequestLogger will be utilised there. In the mean time I've tidied up this log as the existing message is no longer useful.